maze-game/Player.java

```diff
@@ -7,6 +7,7 @@
 public class Player extends Actor
 {
     public static boolean hasWallBreaker = false;
+    public static boolean freeze = false;
```

Comment:
why is this boolean variable needed? why can't we use the same one thats declared in MyWorld?

Reply:
i guess it doesnt really matter for now, as it works